### DISCUSSION TITLE
How to update LXC containers application

### DISCUSSION THREAD

**Discussion by Baptiste-mrch** [tteck/Proxmox#3558] [General]
Hey,

First of all thanks for your script Tteck, it's a really a great and simple way to deploy services !

I'm just questioning myself regarding the update of the application itself. I utilize the CRON LXC Updater, but the application, for my exemple Nginx Proxy Manager, is still in version 2.10.4 whereas the new version is 2.11.3.
I don't really know how LXC container works, so maybe I miss something, but I'm just a little bit lost regarding the CRON LXC Updater and the version of the applications itself. Thanks for enlightening me on this subject !

**Comment by Narkowii** [comment]
just type UPDATE in the LXC shell
it will run the script to update the application

**Reply by Baptiste-mrch** [reply]
Yep this is already what I've tried following discussion #2960 
![image](https://github.com/user-attachments/assets/3140ef0b-7ceb-4941-917a-afe44e64e7a5)

**Comment by Narkowii** [comment]
and if you run the script in the LXC ?
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)"`

**Reply by Baptiste-mrch** [reply]
Wow there is a start, but still I have a fail message
![image](https://github.com/user-attachments/assets/6acf2393-4981-4aa7-8336-238987573d83)
![image](https://github.com/user-attachments/assets/96212d1b-4641-4caa-87f8-945f6ffbd1d7)

**Reply by ucramos** [reply]
i had the same exact error
On the LXC console run "npm install -g pnpm@8.15", then run the update.
